### PR TITLE
Add QSG-1

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/fragments/pickup-manager-saml.md [hunk 1]
```diff
@@ -2,7 +2,70 @@
...
+        "HEAD",
+        "OPTIONS"
+    ]
+    support_any_header = true
+    supported_headers = []
+    exposed_headers = []
+    supports_credentials = true
+    max_age = 3600
+    tag_requests = false
+    ```
```
Comment (dewniMW) [inline en/docs/fragments/pickup-manager-saml.md:29]: use {!fragments/cors-config.md!} instead. Also, let's have this under a separate topic. ex: https://github.com/wso2/docs-is/pull/2506/files#diff-5369a3b41329f57f33c091f53d582ff6181c9664ff60941549e6b98a0bfa5e83R43-R45

File: en/docs/fragments/pickup-manager-saml.md [hunk 2]
```diff
@@ -2,7 +2,70 @@
...
+        3.  **Enable Attribute Profile**
+
+        4.  **Include Attributes in the Response Always**  
+        
+        5.  **Enable Signature Validation in Authentication Requests and Logout Requests**
+            
+    
+    !!! tip
+        For more information on the advanced configurations
+        refer, [Configuring SAML2 WEB Single-Sign-On](../../../guides/login/webapp-saml/)
```
Comment (dewniMW) [inline en/docs/fragments/pickup-manager-saml.md:66]: To configure more advanced configurations, see [SAML Configurations](../../login/saml-app-config-advanced).

File: en/docs/fragments/saml-app-config-basic.md [hunk 3]
```diff
@@ -8,5 +8,5 @@
      
      - The **Assertion Consumer URL** is the Assertion Consumer Service (ACS) URL of the service provider. This URL should be the URL of the page to which the browser is redirected to after successful authentication.
     
-    !!! tip
-        To configure more advanced configurations, see [SAML Configurations](../../login/saml-app-config-advanced). 
+3. To configure more advanced configurations, see [SAML Configurations](../../../guides/login/saml-app-config-advanced).
```
Comment (dewniMW) [inline en/docs/fragments/saml-app-config-basic.md:11]: shouldn't this be a tip?

File: en/mkdocs.yml [hunk 4]
```diff
@@ -67,8 +67,6 @@ nav:
         - 'SAML': quick-starts/webapp-saml-sample.md
         - 'Query SAML Assertions': quick-starts/query-saml-assertions-sample.md
         - 'Use SAML Artifact Binding': quick-starts/use-artifact-binding-sample.md
-        - 'Configure Attributes': quick-starts/attribute-config-sample.md
-        - 'Integrate with a Custom Protocol': quick-starts/webapp-custom-sample.md
```
Comment (dewniMW) [inline en/mkdocs.yml:71]: why are these docs removed?

File: en/mkdocs.yml [hunk 5]
```diff
@@ -79,10 +77,6 @@ nav:
         - 'Overview': quick-starts/single-sign-on.md
         - 'OAuth/OIDC Apps': quick-starts/sso-for-oidc-apps.md
         - 'SAML Apps': quick-starts/sso-for-saml-apps.md
-      - 'Multi-Factor Authentication': 
-        - 'Possession Factor':  
-          - 'SMS OTP': quick-starts/mfa-sample.md
```
Comment (dewniMW) [inline en/mkdocs.yml:84]: why is this removed? this doc is referred in some other places as well

File: en/mkdocs.yml [hunk 6]
```diff
@@ -79,10 +77,6 @@ nav:
...
-          - 'X509': quickstarts/x509-sample.md
```
Comment (dewniMW) [inline en/mkdocs.yml:85]: shouldn't we link the x509.md file here?

File: en/mkdocs.yml (hunk 4 above)
Comment (gomathyk) [reply en/mkdocs.yml:71]: They don't have the attachments. The docs have been written thinking that these files will be available shortly.

Comment (dewniMW) [reply en/mkdocs.yml:71]: in that case, shall we keep it commented instead of removing it so that we don't miss it out in the future?

File: en/mkdocs.yml (hunk 5 above)
Comment (gomathyk) [reply en/mkdocs.yml:84]: Nexmo app is outdated. So this doc needs to be tested with their rebranded version, https://dashboard.nexmo.com/getting-started/sms

File: en/mkdocs.yml (hunk 6 above)
Comment (gomathyk) [reply en/mkdocs.yml:85]: This is not tested with a sample

File: en/mkdocs.yml (hunk 5 above)
Comment (dewniMW) [reply en/mkdocs.yml:84]: since this doc is referred in multiple places, I think it is better if this can be tested. Also, we need this doc as we have a link to this in the landing page (for try it out).

Comment (gomathyk) [reply en/mkdocs.yml:84]: This is referred in the corresponding guide and I have remove the dependencies now. 
Ofcourse, we need to test it but I don't think we should have it in the nav till then.

Comment (dewniMW) [reply en/mkdocs.yml:84]: let's keep it commented

File: en/mkdocs.yml (hunk 6 above)
Comment (dewniMW) [reply en/mkdocs.yml:85]: let's keep it commented

**Review by dewniMW — APPROVED** [review]
